AzureCalling/Controllers/datasource/AudioDeviceSelectionDataSource.swift

```diff
@@ -71,3 +72,8 @@ class AudioDeviceSelectionDataSource: NSObject, BottomDrawerDataSource {
         return cell
     }
 }
+
+//this handles updating the device icon when changed
+protocol AudioDeviceSelectionViewControllerDelegate: AnyObject {
+    func updateDeviceAudioSelection() // updates the device audio icon with current Selection
```

Comment:
Why is this `DeviceAudio` when other places are `audioDevice`. Maybe rename to updateAudioDeviceSelectionButtonIcon or something?